packages/integrations/node/src/middleware.ts

```diff
@@ -2,16 +2,17 @@ import type { NodeApp } from 'astro/app/node';
 import type { IncomingMessage, ServerResponse } from 'http';
 import type { Readable } from 'stream';
 import { responseIterator } from './response-iterator';
+import type { Options } from './types';
 
-export default function (app: NodeApp) {
+export default function (app: NodeApp, options: Options) {
```

Comment:
It would be great if we pass `mode` instead of `options` since we only use the `mode`.

Reply:
> I have one nit but this looks good to me! Could we add some tests too to make sure we don't make regressions in the future?



> It would be great if we pass `mode` instead of `options` since we only use the `mode`.

ok no problem